src/main/kubernetes/artemis/ingresses/activemq-artemis-ingress.yml

```diff
@@ -0,0 +1,20 @@
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  annotations:
+    field.cattle.io/creatorId: user-9jfkn
+  labels:
+    cattle.io/creator: norman
+  name: activemq
+spec:
+  rules:
+  - host: activemq.artemis.rancher.localhost
+    http:
+      paths:
+      - backend:
+          serviceName: activemq-artemis
+          servicePort: 8161
+        pathType: ImplementationSpecific
+  tls:
+  - hosts:
+    - activemq.artemis.rancher.localhost
```

Comment:
This should be target independent

Reply:
Not sure what you mean here, not specifying the host or not including rancher in the host name?

Reply:
I think it would be good to move the hostname configuration to the kustomize file. It is not obvious to the person who deploys this that deep in the configuration files the hostname is hard-coded

Reply:
I undertand now, done in [this](https://github.com/ls1intum/Artemis/pull/4467/commits/c0ec172518a7f1b84cfc5fa764da094e29f57d51) commit